tests/unit-tests/UnitTests/Providers/FileStorage/RedisFileStorageProviderTests.cs

```diff
@@ -0,0 +1,50 @@
+﻿using form_builder.Models;
+using form_builder.Providers.FileStorage;
+using form_builder.Providers.StorageProvider;
+using Moq;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Threading;
+using Xunit;
+
+namespace form_builder_tests.UnitTests.Providers.FileStorage
+{
+    public class RedisFileStorageProviderTests
+    {
+        private readonly RedisFileStorageProvider _redisFileStorageProvider;
+        private readonly Mock<IDistributedCacheWrapper> _mockDistributedCache = new();
+
+        public RedisFileStorageProviderTests()
+        {
+            var formData = JsonConvert.SerializeObject(new FormAnswers { Path = "page-one", Pages = new List<PageAnswers>() });
+
+            _mockDistributedCache.Setup(_ => _.GetString(It.IsAny<string>())).Returns(formData);
+
+            _redisFileStorageProvider = new RedisFileStorageProvider(_mockDistributedCache.Object);
+        }
+
+        [Fact]
+        public void Call_GetString_ReturnCacheAsString()
+        {
+            _redisFileStorageProvider.GetString(It.IsAny<string>());
+
+            _mockDistributedCache.Verify(_ => _.GetString(It.IsAny<string>()), Times.Once);
+        }
+
+        [Fact]
+        public void Call_Remove_CacheData()
+        {
+            _redisFileStorageProvider.Remove(It.IsAny<string>());
+
+            _mockDistributedCache.Verify(_ => _.Remove(It.IsAny<string>()), Times.Once);
+        }
```

Comment:
As above, the test name should reflect the name of the method under test and the expected behaviour/outcome